### PR TITLE
update_code; add test; change some fixtures;

### PR DESCRIPTION
**Pull request by JialiangFan**
I recently write some tests for flask project with peewee. And i think we should DRY. So i find you project  and want to add some fixtures for auto test the repeat rest crud api.
  Hope you can accept my pr. And give me some advises.  Nothing could be better if you cant redevelop this project.

### CODE REVIEW

File: test_fake.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import pendulum
```
Comment (niedbalski) [inline test_fake.py]: Is there a reason for adding a new module? why not use datetime.datetime.now?

**Review by niedbalski — CHANGES_REQUESTED** [review]
First of all, thank you for looking into this project. It's very exciting to have someone actively fixing it and making it better.

I just have a small nitpick for it that you can read in the comments, all the rest looks sane for me.

I'll make sure to add a proper tox/testing target for running your tests under circle-ci , after merging your changes.

File: test_fake.py (hunk 1 above)
Comment (JialiangFan) [reply test_fake.py]: yes. You are right, we should use datetime.

**Comment by niedbalski** [comment]
LGTM

**Pull request merged**